CONTRIBUTING.md

```diff
@@ -36,52 +38,55 @@ Every Issue and PR should be tagged with one or more of the following labels to
 - Comments/sub-Conversations within a PR may only be marked Resolved by:
   - A) The originator of the sub-conversation. 
-  - B) The relevant WSL or 
-  - C) An automated timeout of 48hrs when there has been no additional comments after an update is posted to resolve the sub-thread. 
+  - B) The relevant sub-project lead or 
+  - C) An automated timeout of 1 week when there has been no additional comments after an update is posted to resolve the sub-thread. 
   - **The person responding to a sub-conversation shall not mark it as Resolved**.
 - All changes to an existing PRs shall be made through subsequent commits. 
   - **Do not modify an existing commit, create separate commits under the same PR**.
+  - After major changes to a PR reapproval is needed from all previous approvers
   - PRs will be merged using "Squash and Merge" to enforce a linear commit history.
 
 <a name="approvals"></a>
 ### Pull Requests Approval Process
 
 - Once a PR is created, it needs to get the following approvals before it is merged into master.
-  - From at least one of contributor for that workstream
+  - From at least one of contributor for that sub-project
     - Approvers should be distributed equally among operators and vendors.
-  - From WSL (to make sure consensus is reached).
-  - WSL can request additional approvals reviews.
+  - From sub-project lead (to make sure consensus is reached).
+  - Sub-project lead can request additional approvals reviews.
     - Recommendation is to use 4 approvals for complex PRs.
   - 2 business days cool off period should be applied before Final approval.
-  - Final Approval by TSL (to make sure process is followed)
-- Only One approval will be counted per each organization for a given PR.
-- The selection of which contributor to approve a PR is made by the WSL and should take those factors into consideration:
+  - Only One approval will be counted per each organization for a given PR.
+- The selection of which contributor to approve a PR is made by the sub-project lead and should take those factors into consideration:
   - Contributor needs to be actively discussing the PR to be selected for approval.
-- PRs will be merged automatically online by the TSL once consensus is reached and all approvals are received. 
-- If WSL is the person who is creating the PR, They need to request an alternate approver, preferably from the co-lead or from the contributors list.
+- PRs will be merged automatically online by the sub-project lead or other committers of the sub-projetc once consensus is reached and all approvals are received. 
+  - Sub-projects shold have an agreed set of committers who are trusted with merging rights to the sub-projects. [CODEOWNERS](CODEOWNERS) file should contain all the committers of the sub-projects including the sub-proejct lead.
+- If sub-project lead is the person who is creating the PR, They need to request an alternate approver, preferably from the co-lead or from the contributors list.
```

Comment:
```suggestion
- If sub-project lead is the person who is creating the PR, They need to request an alternate approver, preferably from a co-lead or from the contributors list.
```

I think there could be cases with more than 2 leads, or at least this doesn't "shut that door."